src/hotspot/cpu/riscv/methodHandles_riscv.cpp

```diff
@@ -444,7 +444,6 @@ void MethodHandles::generate_method_handle_dispatch(MacroAssembler* _masm,
       __ far_jump(RuntimeAddress(SharedRuntime::throw_IncompatibleClassChangeError_entry()));
     }
   }
-
```

Comment:
This whitespace deletion seems a bit random, but I have no opinion on it

Reply:
Okey.